### ISSUE TITLE
Don't suppress spaces and wildcards for PostgreSQL FTS use case

### ISSUE THREAD

**Issue by rsignell-usgs** [geopython/pycsw#212]
In the csw:AnyText/apiso:AnyText case, when pycsw is bound to PostgreSQL, PostgreSQL FTS is used (which gives us the search index/engine functionality and performance / scaleability that full text searching requires). 

You need to update the code that handles values like:

```
<ogc:PropertyIsLike wildCard="%" escapeChar="\" singleChar="_">
```

...to pass this to FTS 'as is' instead of the traditional SQL case,and FTS will take care of this intelligently, and results will return.

In other words, don't supressing the spaces and wildcards for the FTS use case, like you currently do:

https://github.com/geopython/pycsw/blob/master/pycsw/fes.py#L165
